pkg/util/helperfunctions.go

```diff
@@ -40,6 +42,19 @@ func GenerateHashFromName(name string) uint64 {
 	return hash.Sum64()
 }
 
+func IsHPAEnabled(jsonReplicas *apiextensionsv1.JSON) bool {
+	if skiperatorv1alpha1.IsReplicasFloat(jsonReplicas) {
+		return false
+	} else if skiperatorv1alpha1.IsReplicasStruct(jsonReplicas) {
```

Comment:
Jeg lurer på om mange av disse `if float else struct`-sjekkene kunne vært `if float else`? Og da kan man også droppe else-klausulen rundt koden under så lenge man gjør en return i if-en over.

```suggestion
	if skiperatorv1alpha1.IsReplicasFloat(jsonReplicas) {
		return false
	}
```